mergify_engine/tests/unit/test_subscription.py

```diff
@@ -174,11 +174,54 @@ async def test_active_feature(redis_cache):
             "features": ["private_repository", "large_repository"],
         },
     )
+
     assert sub.has_feature(subscription.Features.PRIVATE_REPOSITORY) is True
     assert sub.has_feature(subscription.Features.LARGE_REPOSITORY) is True
     assert sub.has_feature(subscription.Features.PRIORITY_QUEUES) is False
 
 
+@pytest.mark.asyncio
+async def test_active_feature(redis_cache):
```

Comment:
This is testing that `has_feature` works so it's a duplicate of the test above.

For this feature, you need a functional test.

Reply:
I will prepare the repository for generating the fixtures